services/server/package.json

```diff
@@ -14,9 +14,14 @@
     "postgres-test:start": "docker-compose -f test/docker-compose.yml up -d",
     "postgres-test:stop": "docker-compose -f test/docker-compose.yml rm -fsv",
     "postgres-test:migrate": "cd ../database && npm run migrate:up -- --env test",
-    "test": "npm run postgres-test:migrate && NODE_CONFIG_DIR=./dist/config DOTENV_CONFIG_PATH=./.env c8 --reporter=none mocha -r dotenv/config --exit",
-    "test-local": "export DOCKER_HOST_POSTGRES_TEST_PORT=${DOCKER_HOST_POSTGRES_TEST_PORT:-5431} && npm run postgres-test:start && echo \"Sleeping 5 seconds to wait for the container to start\" && sleep 5 && npm run postgres-test:migrate && NODE_CONFIG_DIR=./dist/config DOTENV_CONFIG_PATH=./.env c8 --reporter=none mocha -r dotenv/config --exit; status=$?; npm run postgres-test:stop; exit $status",
-    "test:chains": "mocha test/chains/chain-tests.spec.ts --reporter mochawesome --reporter-options reportDir=chain-tests-report,reportFilename=report --exit",
+    "test:unit": "c8 --reporter=none mocha  --exit test/unit/**",
+    "test:integration": "npm run postgres-test:migrate && c8 --reporter=none mocha --exit test/integration/**",
+    "test": "npm run test:unit && npm run test:integration",
```

Comment:
Why did we separate it as :unit and :integration? Just for a way to separate tests or for some technical reason?